src/app/login/login.component.html

```diff
@@ -5,90 +5,62 @@
 
 <!-- Styled login area -->
 <div class="login area">
-  <div class="ui huge white header">Reproducibility Simplified</div>
+  <div class="login banner">
+    <div class="ui huge white header">Reproducibility Simplified</div>
 
-  <div class="ui white header">
-    <p class="nomargin">Tell the Whole Tale about your research through powerful computing tools</p>
-    <p class="nomargin">like Jupyter and RStudio, or craft your own.</p>
+    <div class="ui white header">
+        <p class="nomargin">Empower and share your research with Whole Tale, an open source platform for</p>
+        <p class="nomargin">reproducible computational research. <a href="https://wholetale.org" style="color:white;"><u>Learn more...</u></a></p>
+    </div>
   </div>
 
   <a class="ui large blue button" (click)="loginWithGlobus()">
     <div class="access-wholetale">Access</div>
     <img src="../../assets/img/whole_tale_logo_text_long_light_large.png" width="200px" style="width:200px">
   </a>
 
-  <div class="login icon backdrop">
-    <div class="ui grid">
-      <div class="center aligned four wide column">
-        <i class="huge disk icon"></i>
-        <div class="ui large header">
-          <p class="botmargin">Registered</p>
-          <p class="nomargin">Data</p>
-        </div>
-      </div>
-
-      <div class="two wide column">
-      </div>
-
-      <div class="center aligned four wide column">
-        <i class="huge dashboard icon"></i>
-        <div class="ui large header">
-          <p class="botmargin">Analyze/</p>
-          <p class="nomargin">Compute</p>
-        </div>
-      </div>
-
-      <div class="two wide column">
-      </div>
+  <p style="botmargin" style="color:black;">By accessing Whole Tale, you agree to our
+    <a href="https://docs.wholetale.org/tos" style="color:black"><u>Terms of Use.</u></a>
```

Comment:
Period outside of link:
```suggestion
    <a href="https://docs.wholetale.org/tos" style="color:black"><u>Terms of Use</u></a>.
```